lib/multiply.rb

```diff
@@ -1,2 +1,17 @@
 def multiply(num1, num2)
+    
```

Comment:
Lines 3-15 should all be indented a space. (The shorthand in repl.it and Cloud 9, is `CTRL` + `}`, not `CMD` like I mentioned earlier).